src/components/CheckboxOption.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import {
+  Col,
+  Row,
+  Form,
+} from 'react-bootstrap'
+
+class CheckboxOption extends Component {
+  render() {
+    const option = this.props.options[this.props.optionKey]
+    const model = this.props.model
+
+    return (
+      <Row className="align-items-center pb-2">
+        <Col sm={5}>
+        <Form.Label htmlFor="options-step">
+          {option.title}
+        </Form.Label>
+        </Col>
+
+        <Col sm={7}>
+          <Form.Check
```

Comment:
The other boolean options (like reverse path) are big toggle buttons. What about using one of those instead?

Reply:
I feel that the toggle button should be used only for accordion-reveals of more options. Otherwise it's too heavyweight and disrupts the visual layout of the properties. I like having an "inline", lighter option. That said, I can certainly convert this option to a toggle button.